src/elements/Label/Label.js

```diff
@@ -114,12 +131,6 @@ Label.propTypes = {
   /** Additional text with less emphasis. */
   detail: PropTypes.string,
 
-  /** An element type to render the 'detail' as (string or function). */
-  detailAs: PropTypes.oneOfType([
```

Comment:
So what we'll do there? I suggest leave as is, `detail` prop can be only `number` or `string`.

In #452 we can apply this usage:
```jsx
<Label detail='1 day ago' />                           // string
<Label detail={{ as: 'a', content: '1 day ago' }} />   // props
<Label detail={<img src='1 day ago' />} />             // dom component
<Label detail={<Image src='1 day ago' />} />           // composite component
```